Bukkit/build.gradle.kts

```diff
@@ -101,3 +101,10 @@ tasks.named<ShadowJar>("shadowJar") {
 
     mergeServiceFiles()
 }
+
+tasks {
+    withType<Javadoc> {
+        val opt = options as StandardJavadocDocletOptions
+        opt.links("https://papermc.io/javadocs/paper/1.16")
```

Comment:
It would probably make sense to link the WorldEdit docs too (https://docs.enginehub.org/javadoc/com.sk89q.worldedit/worldedit-core/7.2.0/)